extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/ProbeConfig.java

```diff
@@ -36,6 +36,12 @@ public class ProbeConfig {
     @ConfigItem
     Optional<String> httpActionPath;
 
+    /**
+     * The schema of the {@literal HTTP get} action. Can be either "HTTP" or "HTTPS".
+     */
+    @ConfigItem
+    Optional<String> httpActionSchema;
```

Comment:
Is this correct? Or it was meant to be `httpActionScheme`?

Reply:
Ops, thanks for spotting! 
PR updated